tests/test_utils.py

```diff
@@ -24,17 +25,19 @@ def setup_tmp_list_of_files():
     return tmp_path
 
 
-def setup_zuul_job_yaml():
+def setup_zuul_obj_yaml(obj_type: ZuulObject):
     """Create a temporary directory with a list of files.
 
     Returns
     -------
         A Path object representing the temporary directory.
     """
     tmp_path = pathlib.Path(tempfile.mkdtemp())
-    with pathlib.Path.open(tmp_path / "job.yaml", "w", encoding="utf-8") as f:
+    with pathlib.Path.open(tmp_path / f"{obj_type.value}.yaml", "w", encoding="utf-8") as f:
         f.write(
-            """
+            f"""
+            - {obj_type.value}:
+                name: test-{obj_type.value}
```

Comment:
The pragma object doesn't support the name field.

Reply:
We're not testing the object itself, we're testing the ability to get the objects if they exist. To keep this tests generic, I guess we can keep this, if we need to test the object in the future, we can write directly to a new file